pkg/sdk/logging/model/syslogng/output/redis.go

```diff
@@ -0,0 +1,89 @@
+// Copyright © 2023 Cisco Systems, Inc. and/or its affiliates
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package output
+
+// +name:"Redis"
+// +weight:"200"
+type _hugoRedis interface{} //nolint:deadcode,unused
+
+// +docName:"Sending messages from a local network to the Redis server"
+//
+// ## Prerequisites
+//
+// ## Example
+//
+// {{< highlight yaml >}}
+// apiVersion: logging.banzaicloud.io/v1beta1
+// kind: SyslogNGOutput
+// metadata:
+//
+//	name: redis
+//	namespace: default
+//
+// spec:
+//
+//		redis:
+//		  host: 127.0.0.1
+//		  port: 6379
+//	   retries: 3
+//	   throttle: 0
+//	   time-reopen: 60
+//	   workers: 1
+//
+// {{</ highlight >}}
+type _docRedis interface{} //nolint:deadcode,unused
+
+// +name:"Redis Server Destination"
+// +url:"https://www.syslog-ng.com/technical-documents/doc/syslog-ng-open-source-edition/3.36/administration-guide/49"
```

Comment:
We could use axosyslog docs: https://axoflow.com/docs/axosyslog-core/chapter-destinations/configuring-destinations-redis/